crypto/Pact/Core/Crypto/Base64.hs

```diff
@@ -0,0 +1,36 @@
+module Pact.Core.Crypto.Base64
+  ( encodeBase64UrlUnpadded
+  , decodeBase64UrlUnpadded
+  , fromB64UrlUnpaddedText
+  , toB64UrlUnpaddedText
+  ) where
+
+import Data.Text (Text)
+import Data.Text.Encoding (decodeUtf8)
+import Data.Word
+import Data.ByteString (ByteString)
+
+import qualified Data.ByteString as B
+import qualified Data.ByteString.Base64.URL as B64URL
+import qualified Data.Text as T
+import qualified Data.Text.Encoding as T
+
+equalWord8 :: Word8
+equalWord8 = toEnum $ fromEnum '='
+
+decodeBase64UrlUnpadded :: ByteString -> Either String ByteString
+decodeBase64UrlUnpadded = B64URL.decode . pad
```

Comment:
This error message can fork on the base64 library upgrade

Reply:
It will not, but thanks for the concern! In `Errors.hs`, `evalErrorToBoundedText` handles this and does not print this output.